### DISCUSSION TITLE
[Server Request] Do you guys have added a screen shot features in COD 4?

### DISCUSSION THREAD

**Discussion by ipula** [GameServerManagers/LinuxGSM#3553] [Q&A] [answered]
1. I'm trying to add screen shot feature in my cod4 server.
2.  Is that available in this or any idea how to add to my game server

**Comment by Claiyc** [comment, marked as answer]
I have no idea what a screen shot feature on a server is supposed to look like. A screenshot is definitely something that is entirely client side (in many cases not even related to the game). In fact, you should be able to just press WIN+PRINT in order to create a screenshot.
